### ISSUE TITLE
Make sure issue from KEYCLOAK-17700 is covered

### ISSUE THREAD

**Issue by ssilvert** [keycloak/keycloak-admin-ui#554]
Make sure issue from [KEYCLOAK-17700](https://issues.redhat.com/browse/KEYCLOAK-17700) is covered in the new admin console.

To reproduce in old console:

1. Log in to keycloak Admin Console GUI.
2. Navigate to Clients --> <client> --> Authorization --> Evaluate section.
3. Under Identity Information section, client dropdown does not list all the clients. Dropdown only reaches 20 values.

**Comment by edewit** [comment]
I don't know if it's correct but on the spreadsheet it says that: "This round of design doesn't involve this feature" does that mean we don't need this for 1.0?

**Comment by ssilvert** [comment]
As far as I know, we are doing this for 1.0.

@jenny-s51 You have started on this screen, right?  

FYI for you about this bug in the old UI.

**Comment by jenny-s51** [comment]
Yes @ssilvert I have already implemented this in https://github.com/keycloak/keycloak-admin-ui/pull/1861/files#diff-3b257e0f18de379050bc1a35c07817663c5d83e54026a0573d80976449e64029R119-R160!

I just tested by creating more than 20 clients, and currently they all do get listed in the clients dropdown of the `Authorization -> Evaluate` screen.  [KEYCLOAK-17700](https://issues.redhat.com/browse/KEYCLOAK-17700) seems to have no impact on the functionality here, so it looks like we could probably close this issue.  WDYT @ssilvert? 🙂
